services/ows_refactored/crop_mask/ows_crop_mask_cfg.py

```diff
@@ -0,0 +1,150 @@
+from ows_refactored.common.ows_reslim_cfg import reslim_alos_palsar
+
+bands_crop_mask = {"mask": ["crop_mask", "MASK"], "prob": ["crop_prob", "PROB"]}
+
+
+style_crop_mask_prob = {
+    "name": "prob",
+    "title": "PROB",
+    "abstract": "Prob band",
+    "needed_bands": ["prob"],
+    "index_function": {
+        "function": "datacube_ows.band_utils.single_band",
+        "mapped_bands": True,
+        "kwargs": {
+            "band": "prob",
+        },
+    },
+    "color_ramp": [
+        {"value": 0, "color": "#f7fcf5"},
+        {"value": 750, "color": "#e2f4dd"},
+        {"value": 1000, "color": "#c0e6b9"},
+        {"value": 1500, "color": "#94d390"},
+        {"value": 2500, "color": "#60ba6c"},
+        {"value": 4000, "color": "#329b51"},
+        {"value": 6000, "color": "#0c7835"},
+        {"value": 8000, "color": "#00441b"},
+    ],
+    "legend": {
+        "begin": 0,
+        "end": 8000,
+        "ticks": ["0", "2000", "4000", "6000", "8000"],
+        "tick_labels": {
+            "0": {"label": "0"},
+            "2000": {"label": "2000"},
+            "4000": {"label": "4000"},
+            "6000": {"label": "6000"},
+            "8000": {"prefix": ">"},
+        },
+    },
+}
+
+style_crop_mask_green = {
+    "name": "green",
+    "title": "Green",
+    "abstract": "Classified as water by the decision tree",
+    "value_map": {
+        "crop": [
+            {
+                "title": "Green",
+                "abstract": "Crop",
+                "flags": {"mask": True},
+                "color": "#00ff00",
+            },
+            {
+                "title": "No Crop",
+                "abstract": "No crop",
+                "flags": {"mask": False},
+                "color": "#00ff00",
+                "alpha": 0.0
+            },
```

Comment:
Also, I would use "Crop" and "No Crop" as the titles, and drop the abstract all together.

Will make for a more readable legend.